code/obj/item/device/sacred_heart.dm

```diff
@@ -0,0 +1,58 @@
+#define HEART 1 // I couldn't find a way to make this work without the define, my coding is very primitive however.
```

Comment:
This comment adds no value

Reply:
what exactly do you mean by that? the primitive part, or the entire comment? I added it there because zonespace previously commented on the define being un-needed.